cmd/config/cnv-metrics.yml

```diff
@@ -0,0 +1,124 @@
+---
+# OpenShift virtualization
+# elapsed is the job duration time
+## metadata
+- query: kubevirt_hyperconverged_operator_health_status{}
+  metricName: hco-operator-health
+  instant: true
```

Comment:
I think it could be a good practice to fetch this metric without instant, in order to track HCO's health during the whole workload runtime, another alternative is to add a new alerts-cnv.yml file to track KPIs like this one, find more help about alerts here:

https://kube-burner.github.io/kube-burner/latest/observability/alerting/

and more examples at:

https://github.com/kube-burner/kube-burner-ocp/blob/main/cmd/config/alerts-profiles/alerts.yml